dashboard/src/components/Tabs/Builds/BuildCards.tsx

```diff
@@ -0,0 +1,64 @@
+import { FormattedMessage } from 'react-intl';
+
+import { memo, useMemo } from 'react';
+
+import BaseCard from '@/components/Cards/BaseCard';
+import { GroupedTestStatus } from '@/components/Status/Status';
+
+import type { ISummaryTable } from '../Summary';
+import { DumbSummary, MemoizedSummaryItem } from '../Summary';
+
+interface IErrorsSummaryBuild<T> extends Pick<ISummaryTable, 'summaryBody'> {
+  toggleFilterBySection: (value: string, filterSection: T) => void;
+  diffFilter: Record<string, Record<string, boolean>>;
+}
+
+const ErrorsSummaryBuild = <T,>({
+  summaryBody,
+  toggleFilterBySection,
+  diffFilter,
+}: IErrorsSummaryBuild<T>): JSX.Element => {
+  const summaryHeaders = useMemo(
+    () => [
+      <FormattedMessage key="global.arch" id="global.arch" />,
+      <FormattedMessage key="global.compiler" id="global.compiler" />,
+    ],
+    [],
+  );
+
+  return (
+    <BaseCard
+      title="Summary"
+      content={
+        <DumbSummary summaryHeaders={summaryHeaders}>
+          {summaryBody?.map(row => {
```

Comment:
lets be consistent you are memoizing some arrays like `summaryHeaders` but not others like this one